docs/user/developers/integration.rst

```diff
@@ -161,10 +175,13 @@ InstantSend Status is typically determined through direct connection
 with the dash daemon, `ZMQ notification <https://github.com/dashpay/dash/blob/master/doc/instantsend.md#zmq>`__,
 or through the usage of an external wallet notification script.
 
-**Direct Connection:** InstantSend Status can be identified through
-direct connection with the Dash daemon using JSON-RPC protocol. The
-“instantlock” attribute of the JSON response reflects the status of the
-transaction and is included in the following commands:
+Direct Connection
+^^^^^^^^^^^^^^^^^
+
+InstantSend Status can be identified through direct connection with the Dash
+daemon using JSON-RPC protocol. The ``instantlock`` attribute of the JSON
```

Comment:
instantlock or islock? How is this different to isdlock?

Reply:
isdlock is the p2p message that gets sent on the network. I don't think it ever shows up in rpc responses. If you happen to know that it does let me know.

Reply:
Nope, makes sense to me, thanks for clearing that up for me